samples/crypto-prices/README.md

```diff
@@ -2,6 +2,12 @@
 
 Sequence that keeps printing current crypto prices for a provided pair of currencies every 3 seconds.
 
+___
+
+### The execution of this sample has been visualized in the form of a short video that is available on our [YouTube](https://www.youtube.com/channel/UChgTmKeuAsKj8kDnylkmP6Q) channel [How to check cryptocurrency prices using Scramjet?](https://www.youtube.com/watch?v=BPLKPVVyHNY&t=3s)
```

Comment:
the link it a nice addition. Are we modifying the title as you suggested in the chat? I agree it would sound better in the form of a simple sentence like: 
The video that illustrates the execution of the sample is found on our YouTube.

Reply:
Thank you Alicja, I corrected as you suggested.

Reply:
super, thanks Aga